### PR TITLE
Patrik Mattsson - project 2: News Site

### PR DESCRIPTION
**Pull request by Mattsson717**
Im most proud of making it responsive. 
I struggled with understanding flexbox and grid, what tags to use and when.
Nelify link:
https://nostalgic-fermat-8c63eb.netlify.app/

### CODE REVIEW

File: code/contact.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </nav>
+  </header>
+
+    <section class="big-news">
+            <!-- The big news content -->
+        <img class="contact" src="https://i.ibb.co/pP4jXsH/Screenshot-20210825-231722-Instagram.jpg">
+      <div class="big-text">
+        <h3>CONTACT</h3>
+        <h4>pmattsson89@gmail.com
+        <br>@mattsson717
```
Comment (dandeloid) [inline code/contact.html:34]: <br> for new line of text is supposedly "old school" html and should be avoided. A new line of <p> should to the trick as well maybe?

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,96 @@
...
+      <li class="nav-item">
+        <a href="./contact.html">CONTACT</a>
+      </li>
+      </nav>
     </header>
 
     <section class="big-news">
-      <!-- The big news content -->
+            <!-- The big news content -->
+        <img class="big-image" src="https://media.istockphoto.com/photos/vintage-pacman-video-game-picture-id157739391?s=612x612">
```
Comment (dandeloid) [inline code/index.html:31]: Maybe ad `alt="description"` for images if broken or for visual impaired.

**Review by dandeloid — COMMENTED** [review]
Great job with the project!

The site works! :)
Looks like you fulfilled all the requirements for the Blue, Red and Black flags. It’s responsive and scales well on desktop, tablet and mobile.
You even did some extra pages for the links which is nice.
Good naming of classes and overall understandable code.

Could maybe have some more comments telling what stuff does for extra readability. Some small trimming of code is possible. Made some suggestions in the code, small things.

Overall nice site!

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,182 @@
...
+  justify-content: space-between;
+}
+
+.logo {
+  width: 200px;
+  align-self: center;
+  animation-name: logo;
+  animation: logo 3s ease-in;
+}
+/* Animation */
```
[the remark this answers is not in the document]
Comment (Mattsson717) [reply code/style.css:21]: thank you

**Pull request closed without merging**